pyproject.toml

```diff
@@ -212,6 +213,10 @@ force_grid_wrap = 2
 [tool.flake8]
 max-line-length = 120
 extend-ignore = "E203"
+exclude = [
+    # protobuf compiler output
+    "src/neptune/internal/proto/protobuf_api_series_dto_pb2.py"
```

Comment:
Does "just" `src/neptune/internal/proto/` works? It's probably more future-proof

Reply:
Yep, good catch!